### ISSUE TITLE
Screen Reader Support

### ISSUE THREAD

**Issue by mrdrogdrog** [hedgedoc/react-client#961]
**Is your feature request related to a problem? Please describe.**
Blind or visually impaired people can't use hedgedoc 1.7 in a very useful way. There are keyboard traps, not labeled components and missing instructions.

**Describe the solution you'd like**
- Insert instructions, that are only visible for screenreaders (see bootstrap class `sr-only`)
- Add way more keyboard shortcuts for navigation (switch to editor + switch to output, toolbar shortcuts)
- Add ARIA-tags where possible.

**Additional context**
This should be done as one of the last issues, because the UI structure can still change a lot.
